CRM/Event/Form/SearchEvent.php

```diff
@@ -63,9 +63,7 @@ public function buildQuickForm() {
 
     $event_type = CRM_Core_OptionGroup::values('event_type', FALSE);
 
-    foreach ($event_type as $eventId => $eventName) {
-      $this->addElement('checkbox', "event_type_id[$eventId]", 'Event Type', $eventName);
-    }
+    $this->add('select', 'event_type_id', ts('Event type'), $event_type, FALSE, array('class' => 'crm-select2 huge', 'multiple' => TRUE));
```

Comment:
AddSelect is more compact as it fetches most info from the schema. No need to pass in title, options, etc. Try this:
`$this->addSelect('event_type_id');`
This would let you remove the option group lookup on line 64 too.